localstack-core/localstack/services/kms/provider.py

```diff
@@ -4,10 +4,12 @@
 import logging
 import os
 
+from cbor2._decoder import loads as cbor2_loads
```

Comment:
Rather than using an internal method `_decoder` here we should use the following `from cbor2 import loads`.